client.go

```diff
@@ -183,6 +184,20 @@ func (c *Client) PushWithContext(ctx Context, n *Notification) (*Response, error
 
 	setHeaders(request, n)
 
+	log.Println("Request url:")
```

Comment:
rather than use `log.Println`, it would be safer to use `slog.Info` or `slog.Debug` - that way, it's not critically important that this code not go to prod - by using one of these methods, it could go to prod (where logging is configured to only happen at Warn and higher) and the only impact would be the performance cost of stringification/json marshalling, rather than also having the cost of spamming datadog as well.  If we really wanted to get fancy we could even check `slog.Default().Enabled(context.Background(), slog.LevelDebug)` first and avoid the stringification/json marshalling cost too

Reply:
> having read the apns-push-service pull, I think you've basically already done this, and this pull was useful for review but will eventually be closable without merge
Yes, that's what I meant in the PR overview. Will adopt the logging level suggestion👍